peps/pep-0771.rst

```diff
@@ -8,6 +8,7 @@ Type: Standards Track
 Topic: Packaging
 Created: 13-Jan-2025
 Post-History:
+  `09-Jun-2025 <https://discuss.python.org/t/94905/>`__,
   `15-Jan-2025 <https://discuss.python.org/t/77892/>`__,
   `06-Feb-2025 <https://discuss.python.org/t/79706/>`__,
```

Comment:
Please put this last in the list, and also update `Discussions-To`.
```suggestion
  `15-Jan-2025 <https://discuss.python.org/t/77892/>`__,
  `06-Feb-2025 <https://discuss.python.org/t/79706/>`__,
  `09-Jun-2025 <https://discuss.python.org/t/94905/>`__,
```

Reply:
Oops, not enough coffee